cinema/views.py

```diff
@@ -32,27 +45,108 @@ class CinemaHallViewSet(viewsets.ModelViewSet):
 
 class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
-    serializer_class = MovieSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+
+        if actors:
+            actors = actors.split(",")
+            queryset = queryset.filter(actors__id__in=actors)
```

Comment:
this will work because Django covers for us here but remember that ids are usually ints (if this is what is used in the db table)

Reply:
Thank you! I'll keep it in mind.